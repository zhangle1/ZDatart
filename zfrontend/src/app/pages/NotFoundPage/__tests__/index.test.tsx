import React from 'react';
import { HelmetProvider } from 'react-helmet-async';
import { MemoryRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import { NotFoundPage } from '..';

const renderPage = () =>
  renderer.create(
    <MemoryRouter>
      <HelmetProvider>
        <NotFoundPage />
      </HelmetProvider>
    </MemoryRouter>,
  );

describe('<NotFoundPage />', () => {
  // TODO(Owner): fix app tests...
  it.skip('should match snapshot', () => {
    const notFoundPage = renderPage();
    expect(notFoundPage.toJSON()).toMatchSnapshot();
  });

  it('should should contain Link', () => {});
});
