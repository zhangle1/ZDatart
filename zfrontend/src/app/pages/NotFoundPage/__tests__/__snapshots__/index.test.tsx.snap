// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFoundPage /> should match snapshot 1`] = `
<h2>
  Page not found.
</h2>
`;
